restore/restore.go

```diff
@@ -129,12 +184,16 @@ func (c *RestoreCommand) StartRestore(cliConnection plugin.CliConnection, servic
 		fmt.Println(AddColor("FAILED", red))
 		var message string = string(body)
 		var parts []string = strings.Split(message, ":")
```

Comment:
better to check the size and then access third element of array right ? what if the response from the API changes , like it might already if we merge the broker code ?

Reply:
Done.